aten/src/ATen/Declarations.cwrap

```diff
@@ -3801,6 +3801,25 @@
     - arg: double lambd
       default: 1
 ]]
+[[
+  name: random_gamma_
+  types:
+    - floating_point
+  backends:
+    - CPU
+    - CUDA
```

Comment:
I might be missing something, but I have the impression that we don't have THC code for `random_gamma_`, is that right?